gatsby-site/src/components/footer/Footer.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react'
+import Link from 'gatsby-link'
+import Mailto from 'react-mailto'
+import "../Navbar/navbar.css"
+import "./Footer.css"
+
+
+export default function Footer(props) {
+    const footerIcons = [
+    {
+      imageUrl: 'https://res.cloudinary.com/colark/image/upload/v1535557446/Colark%20Marketing%20Site/email2.svg.svg',
+      linkUrl: 'ellie@colark.com',
+      alt: 'Email Colark'
+    },
+    {
+      imageUrl: 'https://res.cloudinary.com/colark/image/upload/v1535501727/Colark%20Marketing%20Site/Twitter.svg',
+      linkUrl: 'https://twitter.com/colarkinc',
+      alt: 'Find Colark on Twitter'
+    },
+    {
+      imageUrl: 'https://res.cloudinary.com/colark/image/upload/v1535501727/Colark%20Marketing%20Site/LINKEDIN.svg',
+      linkUrl: 'https://www.linkedin.com/company/colark/',
+      alt: 'Find Colark on LinkedIn'
+    },
+    {
+      imageUrl: 'https://res.cloudinary.com/colark/image/upload/v1535501726/Colark%20Marketing%20Site/Facebook.svg',
+      linkUrl: 'https://www.facebook.com/colarkinc/',
+      alt: 'Find Colark on Facebook'
+    }];
+
+    const footerIconList =
+      footerIcons.map((icon, index) => {
+        if (icon.linkUrl.includes("@")) {
+          return (<Mailto key={index} email={icon.linkUrl}>
+            <img
+            src={icon.imageUrl}
+            alt={icon.alt}
+            className="footer__icons--icon-image"/>
+          </Mailto>);
+        } else {
+          return (<a key={index} href={icon.linkUrl} target="_blank">
+            <img
+            src={icon.imageUrl}
+            alt={icon.alt}
+            className="footer__icons--icon-image"/>
+          </a>);
+      }
+    });
+
+    return (
+      <div className="footer colark-footer">
+        <div className="footer__contents">
+          <svg className="footer-logo" width="145" height="50" viewBox="0 0 145 50" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
I forget if this was a feature we wanted, it isn't on Jira, but do we want to make the logo clickable like in the Navbar?